dev/mimicry/config.json

```diff
@@ -974,7 +974,7 @@
                 ]
             }
         },
-        "/hsm/v1/Inventory/EthernetInterfaces?type=Node": {
+        "/hsm/v2/Inventory/EthernetInterfaces?type=Node": {
```

Comment:
Has all of BSS been converted to hsm/v2? The EthernetInterfaces endpoint is one of the few actual endpoint differences between v1 and v2.

Reply:
Also this example payload is in the wrong format

Reply:
Looks like it might have been already: https://github.com/Cray-HPE/hms-bss/blob/master/cmd/boot-script-service/sm.go#L137